controllers/servers/csi/addons_server.py

```diff
@@ -18,62 +20,54 @@ class ReplicationControllerServicer(pb2_grpc.ControllerServicer):
 
     @csi_method(error_response_type=pb2.EnableVolumeReplicationResponse, lock_request_attribute="volume_id")
     def EnableVolumeReplication(self, request, context):
-        utils.validate_addons_request(request)
+        replication_type = self._get_replication_type(request)
+        utils.validate_addons_request(request, replication_type)
 
         volume_id_info = utils.get_volume_id_info(request.volume_id)
         volume_id = volume_id_info.ids.uid
-        volume_internal_id = volume_id_info.ids.internal_id
-
-        other_volume_id_info = utils.get_volume_id_info(request.replication_id)
-        other_volume_internal_id = other_volume_id_info.ids.internal_id
-
-        other_system_id = request.parameters.get(servers_settings.PARAMETERS_SYSTEM_ID)
-        copy_type = request.parameters.get(servers_settings.PARAMETERS_COPY_TYPE, REPLICATION_DEFAULT_COPY_TYPE)
+        rep_request = self._generate_replication_request(request, replication_type)
```

Comment:
`replication_request`? same in all methods

Reply:
This replication request is used inside get_replication function of mediator. And it is the same

Reply:
I think we should change its name to `replication_request`

Reply:
same in all methods

Reply:
Done